Unix/oi/gen_c/common/OIParser.c

```diff
@@ -621,15 +621,15 @@ static MI_Boolean _ParseFunctionDecl(_In_z_ char * line, _In_ OIDefaults * defau
     line2 = _Trim(line);
     if (_StartsWith(line2, "void") == 0)
     {
-        OIERROR1("Trace declaration function didnt start with void! [%s]", line2);
+        OIERROR1("Trace declaration function didn't start with void! [%s]", line2);
```

Comment:
I'm not actively enforcing `'`s, (for smaller things I sometimes offer it), but this felt quasi user-facing.